metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "netmanagers-dnsmasq",
```

Comment:
Possibly - Though I think that will make it a separate module on Puppetforge, which might confuse existing users